cranelift/codegen/src/machinst/lower.rs

```diff
@@ -1077,6 +1086,12 @@ impl<'func, I: VCodeInst> Lower<'func, I> {
             self.finish_bb();
         }
 
+        // Check for any deferred vreg-temp allocation errors, and
+        // bubble one up at this time if it exists.
+        if let Some(e) = self.vreg_alloc_error {
+            return Err(e);
+        }
```

Comment:
Does it make sense to check this after lowering each block rather than only after lowering all blocks? Not saying this isn't sufficient, just wondering whether it makes sense to potentially notice these errors and propagate them earlier.

Reply:
Good point, lifted this into the loop.